packages/core/src/components/icon/icon.stories.ts

```diff
@@ -15,19 +15,28 @@ export default {
         type: { summary: colors.map((value) => `"${value}"`).join('|') },
       },
     },
+    name: {},
```

Comment:
What does `name` do here?

Reply:
Simply reorders it above `aria-` attributes

Reply:
I see.

What if it's:

```suggestion
    name: {
      table: { type: { summary: 'string' } },
    },
```

Just less confusing IMO...

Reply:
What about this? 😅 

```suggestion
    name: {}, // just reorders in the table
```

Reply:
Fine, also TBH can also miss the ordering, I don't mind it.

Reply:
No ordering is weird, `color -> aria-hidden -> aria-label -> name`.

Arguably, `name` is the most important.

I'll leave as is with the comment for now, we can review if/when we do MDX.